includes/class-database-cleanup.php

```
@@ -605,6 +609,155 @@ public static function clean_orphan_postmeta() {
+				$name = isset( $row['option_name'] ) ? (string) $row['option_name'] : '';
+				if ( '' === $name ) {
+					continue;
+				}
+				$result[] = array(
+					'option_name' => $name,
+					'size'        => (int) ( $row['opt_size'] ?? 0 ),
+				);
+			}
+
+			return $result;
+		}
```

PHPMD (2.15.0)

[error] The method get_autoloaded_options is not named in camelCase. (undefined)

(CamelCaseMethodName)